examples/workflows/aws_privatelink/main.tf

```diff
@@ -80,7 +84,7 @@ provider "aws" {
   region = var.aws_region
 }
 
-resource "cockroach_cluster" "cockroach" {
+resource "cockroach_cluster" "example" {
```

Comment:
"example" isn't bad.  The naming convention link I shared suggests naming it "this" if there is only one of a thing.  Other ideas for naming here are "dedicated"  or "serverless" for the cluster.  In real life for a sql_user I'd probably name it something like "admin" or "schema_owner" or "readonly" or "Evan" but example seems fine too.  If you don't feel inspired by any of these ideas I'm fine keeping it example everywhere.